src/components/NavigationIcon/NavigationIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
+import './NavigationIcon.scss';
+
+interface NavigationIconLinkProps {
+  Icon: OverridableComponent<any>;
+  title: string;
+}
+
+export const NavigationIcon: React.FC<NavigationIconLinkProps> = ({ Icon, title }) => (
+  <div className="navigationIcon">
+    <Icon className="navigationIcon__icon" />
+    <h3 className="navigationIcon__title">{title}</h3>
```

Comment:
it not correct becuase its reusable component and it doesn't know where will be used (Use Tab on your navigation items and check how it behaves). I mean using h3 is something wrong here. span or p will be better but not h3 (remember in page there is h1, h2 and in some pages it will be threated as h2 - like +- subtitle)